app/elements/company/challenge/challenge-detail.html

```diff
@@ -60,7 +54,19 @@ <h1 id="challengeName">{{challenge.name}}</h1>
 				</paper-material>
 
 				<paper-material class="card">
-					<h3>Invited users</h3>
+					<div class="layout-horizontal">
+						<h3 class="flex">Invited users</h3>
+						<paper-button on-click="inviteCandidates">
+							<iron-icon icon="{{inviteIcon}}"></iron-icon>
+							Invite candidates
+						</paper-button>
+					</div>
+					<invite-candidates
+						id="inviteCandidates"
+						style="display: none"
```

Comment:
i put inline style to avoid having another `inviteCandidatesStyle` property with default value.